root.store.js

```diff
@@ -22,7 +31,20 @@ const getEnhancers = () => {
   return enhancers;
 };
 
-export const configureStore = createStore(
-  rootReducer,
-  compose(getMiddleware(), ...getEnhancers())
-);
+
+export default function configureStore() {
+  const store = createStore(rootReducer, compose(getMiddleware(), ...getEnhancers()));
+
+  if (module.hot) {
```

Comment:
I copied these redux devtools related code from my own project, but it seems unnecessary here, so I removed.